css/mpl_widget.css

```diff
@@ -18,9 +24,12 @@
 /* Figure */
 
 .jupyter-matplotlib-figure {
-    width: auto;
+    width: fit-content;
     height: auto;
     overflow: hidden;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
@martinRenou i thought the centering was reverted?

Reply:
It was.. This looks like a leftover. But this does not seem to affect the plot? It is not centered when I try latest.

Reply:
huh i was just seeing this behavior. let me try in a fresh env

Reply:
I see this on the binder link - maybe this is browser based. I'm on Firefox 93

Reply:
indeed on chrome it is not centered, but it is on firefox:

left = chrome
right = firefox
![Peek 2022-01-19 18-15](https://user-images.githubusercontent.com/10111092/150233683-93d9a8df-5452-4e23-af45-d17ddff94061.gif)

Reply:
I wish the visual regression tests caught that. We might be able to use playwright with Firefox as well. #422

Reply:
Thanks a lot for catching this!